File: backend/facade/siwatch-device-api/src/main/java/ru/nord/siwatch/backend/facade/device/models/profile/MonitoringPolicy.java
package ru.nord.siwatch.backend.facade.device.models.profile;

import lombok.Getter;
import lombok.Setter;

import java.time.Duration;
import java.util.List;

@Getter @Setter
public class MonitoringPolicy
{
    private int minBatteryLevel;
    private List<MonitorConfig> monitors;
    private Duration syncInterval;
    private int packetSize;
}

File: watch-app/SiWatchApp/Configuration/MonitoringPolicy.cs
using System;
using System.Collections.Generic;
using Newtonsoft.Json;

namespace SiWatchApp.Configuration
{
    public class MonitoringPolicy
    {
        public int MinBatteryLevel { get; set; }

        public List<MonitorConfig> Monitors { get; set; }

        public int SyncInterval { get; set; }

        public int PacketSize { get; set; }

        public override string ToString()
        {
            return $"MonitoringPolicy{{MinBatteryLevel={MinBatteryLevel},SyncInterval={SyncInterval},PacketSize={PacketSize},Monitors=[{String.Join(",", Monitors)}]}}";
        }
    }
}
